saphanabootstrap-formula.spec

```diff
@@ -44,13 +55,40 @@ SAP HANA deployment salt formula
 pwd
 mkdir -p %{buildroot}/srv/salt/
 cp -R %{fname} %{buildroot}/srv/salt/%{fname}
+cp -R %{ftemplates} %{buildroot}/srv/salt/%{fname}/%{ftemplates}
+
+# SUMA Specific
+mkdir -p %{buildroot}/usr/share/susemanager/formulas/states/%{fname}
+mkdir -p %{buildroot}/usr/share/susemanager/formulas/metadata/%{fname}
+cp -R %{fname} %{buildroot}/usr/share/susemanager/formulas/states/%{fname}
+cp -R %{ftemplates} %{buildroot}/usr/share/susemanager/formulas/states/%{fname}/%{ftemplates}
```

Comment:
@arbulu89 I think it is missing the directory creation for the templates directory.

Reply:
@diegoakechi I don't think this is really needed. The recursive copy creates the folder too